packages/core/src/lib/sign-in-experience.test.ts

```diff
@@ -51,26 +60,66 @@ describe('validate terms of use', () => {
 
 describe('validate sign-in methods', () => {
   describe('There must be one and only one primary sign-in method.', () => {
-    test('should throw when there is no primary sign-in method', async () => {
+    test('should throw when there is no primary sign-in method', () => {
```

Comment:
i think that's the point of `it()`: `it should throw...`
doesn't matter though

Reply:
Get it. Will improve it in another PR later.